src/v3/src/components/AuthCoin/authCoinConfigUtil.ts

```diff
@@ -42,7 +42,7 @@ type AuthCoinConfig = {
   iconClassName: string;
 };
 
-const AuthCoinByAuthenticatorKey: Record<string, AuthCoinConfig> = {
+export const getAuthCoinConfiguration = (): Record<string, AuthCoinConfig> => ({
```

Comment:
Why do we need to convert this to a function? Or is this basically the fix?

Reply:
Ok, that makes sense, thanks. Any thoughts on storing description as the i18n key and calling `loc` at the point of usage?

Reply:
:D I actually did consider that, the problem with that was since the authcoin is also used by Authenticator buttons, those contain their own localized descriptions which are translated within the transformers :( 

I considered adding a `key` property for this exact scenario, but then I would have to add logic to check if the key exists and use that otherwise use description but thought that would complicate things more :(